ocaml/src/cli_asd.ml

```diff
@@ -365,6 +365,100 @@ let osd_bench_cmd =
   in
   osd_bench_t, info
 
+let asd_multistatistics long_ids to_json verbose cfg_file tls_config clear =
+  begin
+    let process_results results =
+      if to_json
+      then
+        begin
+          let result_to_json rs =
+            let x  =
+              List.map
+                (fun (long_id, statso) ->
+                  match statso with
+                  | None       -> long_id, `Null
+                  | Some stats -> long_id, Alba_json.AsdStatistics.to_yojson stats
```

Comment:
It may make sense to wrap this result within a Alba_json.Result.t ?
That way we can later on - if desired - provide more info about which error happened while getting the statistics for a certain asd.